samples/bot-tag-mention/csharp/AppManifest/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.14",
```

Comment:
Please check if we can use latest version of manifest.

Reply:
fixed